client/js/components/user/DpOrganisationList/DpOrganisationFormFields.vue

```diff
@@ -590,19 +590,13 @@
           <p class="font-size-6 weight--normal">
             {{ Translator.trans('explanation.organisation.copies.paper') }}
           </p>
-          <select
-            class="bg-color--white"
-            style="height: 27px;"
+          <dp-select
```

Comment:
While DpSelect can also be used to render a standalone select element, it's primary use is to produce a select element alongside the label (and, if needed, hint text) for a form element. To achieve this, the values from line 589 (and 591 for the hint) can be passed into the `label` prop of DpSelect: 
```
<dp-select
  :label="{ text: yourLabelText, hint: yourLabelHint }"
  ... />
```